python/paddle/fluid/layers/nn.py

```diff
@@ -10652,3 +10652,60 @@ def tree_conv(nodes_vector,
     else:
         pre_activation = out
     return helper.append_activation(pre_activation)
+
+
+from .ops import square
+from .control_flow import equal
```

Comment:
this usually mean the follow layer doesn't belong to this file or some refactoring is needed.